package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@ant-design/icons": "^4.0.5",
+    "@ant-design/icons": "^4.2.2",
     "@nivo/bar": "^0.61.1",
-    "antd": "^4.0.4",
+    "antd": "^4.6.1",
```

Comment:
Updated to improve search box accessibility, the role went from `generic` to `textbox`.